src/olympia/scanners/models.py

```diff
@@ -1,44 +1,47 @@
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
 from django_extensions.db.fields.json import JSONField
 
 from olympia.amo.models import ModelBase
-from olympia.constants.scanners import SCANNERS
+from olympia.constants.scanners import SCANNERS, ACTIONS, NO_ACTION
 from olympia.files.models import FileUpload
 
 
 class ScannerResult(ModelBase):
-    upload = models.ForeignKey(FileUpload,
-                               related_name='scanners_results',
-                               on_delete=models.SET_NULL,
-                               null=True)
+    upload = models.ForeignKey(
+        FileUpload,
+        related_name='scanners_results',
+        on_delete=models.SET_NULL,
+        null=True,
+    )
     # Store the "raw" results of a scanner (optionally).
     results = JSONField(default=None)
     scanner = models.PositiveSmallIntegerField(choices=SCANNERS.items())
-    version = models.ForeignKey('versions.Version',
-                                related_name='scanners_results',
-                                on_delete=models.CASCADE,
-                                null=True)
+    version = models.ForeignKey(
+        'versions.Version',
+        related_name='scanners_results',
+        on_delete=models.CASCADE,
+        null=True,
+    )
     matches = JSONField(default=[])
     has_matches = models.NullBooleanField()
 
     class Meta:
         db_table = 'scanners_results'
         constraints = [
-            models.UniqueConstraint(fields=('upload', 'scanner', 'version'),
-                                    name='scanners_results_upload_id_scanner_'
-                                         'version_id_ad9eb8a6_uniq'),
-        ]
-        indexes = [
-            models.Index(fields=('has_matches',)),
+            models.UniqueConstraint(
+                fields=('upload', 'scanner', 'version'),
+                name='scanners_results_upload_id_scanner_'
+                'version_id_ad9eb8a6_uniq',
+            )
         ]
+        indexes = [models.Index(fields=('has_matches',))]
```

Comment:
I have a `Black` vim plugin to autoformat my code (because I am missing prettier all the time) 😞